src/styles/theme.js

```diff
@@ -2,14 +2,16 @@
 import React from 'react';
 import type { Context } from 'react';
 
+import type { ThemeName } from '../reduxTypes';
+
 export type ThemeColors = {|
   color: string,
   backgroundColor: string,
   cardColor: string,
   dividerColor: string,
 |};
 
-export const themeColors: { [string]: ThemeColors } = {
+export const themeColors: { [name: ThemeName | 'light']: ThemeColors } = {
```

Comment:
Happy to have this be made more specific, but I was a bit confused by the commit message.

* Instead of "fix annotation", the message can be more specific: namely that it makes the type (or the annotation) more specific.

* I kept wanting to read the paragraph in the body as an explanation of the "why" for this change. But I couldn't see the connection -- and now I think the idea is that it's describing a future change we probably want to make to this code. Is that right?

Reply:
> Instead of "fix annotation", the message can be more specific: namely that it makes the type (or the annotation) more specific.

~~I think the "fix" was aimed at `[string]`, which looks like a subtle mistake (though not a large one) — `string` isn't interpreted as a type; it's just what we were calling the indexer property.~~ Huh, looks like I was wrong about this! ([doc](https://flow.org/en/docs/types/objects/#toc-objects-as-maps))

Makes sense.

> I kept wanting to read the paragraph in the body as an explanation of the "why" for this change. But I couldn't see the connection -- and now I think the idea is that it's describing a future change we probably want to make to this code. Is that right?

Mm, yeah (that's right), I'll make that clearer.

Reply:
In fact, maybe I'll move it to a code comment on the `ThemeName` type declaration in `src/reduxTypes`.